src/Compressors/GzipCompression.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Spatie\DbDumper\Compressors;
+
+class GzipCompression implements Compression
```

Comment:
Let's get the naming just right and rename this class to `GzipCompressor` and the interface to `Compressor` so that it is more in line with the namespace.

Reply:
Done!